docs/src/sv_operator/sv_operations.rst

```diff
@@ -263,3 +263,101 @@ Upper bound
+(up until the fundamental limitations behind this issue have been resolved in Splice; expected with the upgrade to Canton 3.4):
+
+1. Shortly before the Daml upgrade becomes effective:
+   Each SV pauses their ``ExpireRewardCouponsTrigger`` trigger by extending ``.additionalEnvVars`` in ``sv-values.yaml`` in the following way:
+
+   .. code-block:: yaml
+
+      additionalEnvVars:
+        - name: ADDITIONAL_CONFIG_PAUSED_EXPIRY_TRIGGER
+          value: |
+            canton.sv-apps.sv.automation.paused-triggers += "org.lfdecentralizedtrust.splice.sv.automation.delegatebased.ExpireRewardCouponsTrigger"
+
+2. Shortly after the Daml upgrade becomes effective:
+   One SV operator :ref:`constructs a party exclusion config <sv_ops_ignored_rewards_party_ids_determine>`,
+   confirms its correctness and shares it with the group of all operators.
+3. Each SV operator adopts the party exclusion config and resume the ``ExpireRewardCouponsTrigger`` trigger by reverting the change from step 1.
+   If ``.additionalEnvVars`` in ``sv-values.yaml`` was empty before step 1, it is now expected to look similar to the following:
+
+   .. code-block:: yaml
+
+      additionalEnvVars:
+        - name: ADDITIONAL_CONFIG_IGNORED_EXPIRY_PARTY_IDS
+          value: |
+            canton.sv-apps.sv {
+              automation.ignored-expired-rewards-party-ids = [ "party1::12345", "party2:57890" ]
+            }
+
+Updates to the party exclusion config are possible at future points in time,
+e.g., in case validators that were previously outdated eventually upgrade to a sufficiently recent Splice version.
+
+.. _sv_ops_ignored_rewards_party_ids_determine:
+
+Determining parties to ignore
++++++++++++++++++++++++++++++
+
+The process to determine the right parties to ignore is advanced and likely to involve some trial-and-error.
+It is sufficient for a single operator to perform this work and arrive at a suitable config.
+
+In the following we list complimentary approaches for building the list of parties.
+
+Prerequisite steps:
+
+1. Wait until the new Daml upgrade has already become effective.
+2. Pick a round that was open shortly after the Daml upgrade became effective (to aid database queries); we will denote this round as ``UPGRADE_ROUND`` in the following.
+3. Resume the expiry trigger (accepting that it will fail until you have completed the ignore list).
+4. Set ``delegatelessAutomationExpiredRewardCouponBatchSize: 1`` in your ``sv-values.yaml``.
+
+Based on that:
+
+- You can wait until the trigger fails, then investigate which parties are involved in the failing transaction and add these to the set of parties to ignore.
+- For a more quick feedback loop, you can attempt queries against the SV app (postgres) database.
+  For example, you can list receivers of rewards that were not expired since the Daml upgrade:
+
+  .. code-block:: sql
+
+      select distinct(reward_party) from dso_acs_store where reward_round < UPGRADE_ROUND;
```

Comment:
Note that this overapproximates: It could also be that the trigger just hasn't gotten to those contracts yet. So I'm a bit wary of documenting that mess.